crates/ark/src/help/r_help.rs

```diff
@@ -279,7 +279,15 @@ impl RHelp {
                 RObject::from(R_GlobalEnv)
             })();
 
-            let obj = harp::parse_eval0(topic.as_str(), env.sexp)?;
+            let obj = match harp::parse_eval0(topic.as_str(), env.sexp) {
+                Ok(obj) => obj,
+                Err(err) => {
+                    // Could not parse/eval the topic; no custom handler.
+                    log::trace!("Could not parse/eval help topic expression '{}': {:?}", topic, err);
```

Comment:
Probably should be a `warn` so users can figure out why their gesture doesn't work?

```suggestion
                    log::warn!("Could not parse/eval help topic expression '{}': {:?}", topic, err);
```

I'm on the fence regarding whether it should be a visible notification as in the linked issue, but conservative logging seems like a good first approach.